stl/inc/xstring

```diff
@@ -3807,12 +3807,13 @@ public:
             // either we are shrinking, or the growth fits
             // may temporarily overflow; OK because size_type must be unsigned
             const auto _New_size = _Old_size + _Count - _Nx;
-            _ASAN_STRING_MODIFY(*this, _Old_size, _New_size);
+            _ASAN_STRING_REMOVE(*this);
             _Mypair._Myval2._Mysize = _New_size;

```

Comment:
No change requested: We could fuse `_New_size` now, or move its definition below the `_ASAN_STRING_REMOVE`. This is fine as-is though.